src/test/scala/mesosphere/marathon/MarathonSchedulerActorTest.scala

```diff
@@ -140,7 +139,6 @@ class MarathonSchedulerActorTest extends AkkaUnitTest with ImplicitSender with G
         .addTaskGone(containerName = Some("gone"))
         .addTaskDropped(containerName = Some("dropped"))
         .addTaskUnknown(containerName = Some("unknown"))
-        .addTaskReserved(containerName = Some("reserved"))
```

Comment:
It's not clear to me why this line was removed

Reply:
Alena explained this to me; sounds like the intention was to remove reserved condition completely, but this ended up being too large. We may need to add this line back, or drop reserved completely.

Reply:
done

Reply:
Sorry, @alenkacz, I reverted b74d2cf0d5bbdeba36f2f0807e643f9b1d693024 to have this branch build. Please create a pull request into this branch so we keep this branch green.

Reply:
Is there any other test we have to bring back?